feat: add css colors lab

ihechikara opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #395

<!-- Feel free to add any additional description of changes below this line -->

jdwilkin4 commented on frontend-cert/css-projects/css-colors/user-stories.md:
```diff
@@ -0,0 +1,25 @@
+
+2. You should have a `div` with a class of `color-grid`.
+
+3. You should have five `div` elements within `div` created above.
```
I would reference the class here

```suggestion
3. You should have five `div` elements within the `.color-grid` `div`.
```

jdwilkin4 reviewed (commented):
Left a small comment 👍

moT01 commented on frontend-cert/css-projects/css-colors/user-stories.md:
```diff
@@ -0,0 +1,25 @@
+
+3. You should have five `div` elements within `div` created above.
+
+4. The first `div` should have these classes: `color-box color1`. 
```
This is looking good @ihechikara 👍 

Here's some slight adjustments to the user stories that make it a little more specific for some things, but also more open-ended for some of the colors:

You should set the background color for `body` to `#f4f4f4`.
You should have a `div` with a class of `color-grid` to hold all your color elements.
You should have five `div` elements within the `.color-grid` element.
The five `div` elements should each have a class of `color-box` and `color#`, where `#` is the number of the order of that `div`. For example, `color1` for the first `div`, and `color2` for the second, etc.
All of your `.color-box` elements should have a set `width` and `height` so they are visible on the page.
The `.color1` element should have a `background-color` that uses a predefined (word) color value.
The `.color2` element should have a `background-color` that uses a hexadecimal color value.
The `.color3` element should have a `background-color` that uses an RGB color value.
The `.color4` element should have a `background-color` that uses a HSL color value.
The `.color5` element should have a `background-color` set.

I changed it so that one of each type of color values is required, but not defined - so users can pick the colors, but they have to use all types.

Also note that if we change the user stories to be more open-ended like this, the prototype won't fulfill them anymore.

Let me know what you think @ihechikara.

ihechikara replied:
Thanks for the suggestions, @moT01! I've updated the PR - let me know what you think

jdwilkin4 commented on frontend-cert/css-projects/css-colors/index.html:
```diff
@@ -0,0 +1,19 @@
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>CSS Colors Lab</title>
```
I would call this something colored boxes

```suggestion
    <title>Colored Boxes</title>
```

jdwilkin4 commented on frontend-cert/css-projects/css-colors/index.html:
```diff
@@ -0,0 +1,19 @@
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <h1>CSS Colors</h1>
```
```suggestion
    <h1>Colored Boxes</h1>
```

Dario-DC commented on frontend-cert/css-projects/css-colors/styles.css:
```diff
@@ -0,0 +1,50 @@
+}
+
+.color-box {
+    display: flex;
```
I'm afraid FlexBox has not been taught yet by this point.

Dario-DC replied:
But now I see that it's not required :+1:

moT01 reviewed (changes requested):
Here's some more suggestions @ihechikara...

When I run your code, I don't see anything. The prototype is supposed to fulfill the user stories. Here's how you can do that:

Give each of the `.color#` class a background color described by the user stories. e.g. hex, hsl, etc. Give each `.color-box` a width and height so they're visible on the page.

ihechikara commented:
I've added them back, @moT01.

> Also note that if we change the user stories to be more open-ended like this, the prototype won't fulfill them anymore.

I removed them initially because in your last suggestion👆 you mentioned that having the user stories more open-ended meant that the prototype won't fulfil the stories. I may have misunderstood that. 

Let me know what you think!

moT01 reviewed (approved):
LGTM @ihechikara 🎉
